kotlin-code-style.md

```diff
@@ -36,48 +40,49 @@ class Person(val firstName: String, val lastName: String, var age: Int) {
 }
 ```
 
-#Extensions
+# Extensions
 
-I suggest we create one extension file for each Type we need :
+When adding extensions to external classes, make separate files for each type
+and name them _<Type>Extensions.kt_:
 
 ```
-- StringExtension.kt
-- BitmapExtension.kt
-- ContextExtension.kt
-...
+- StringExtensions.kt
+- BitmapExtensions.kt
+- ContextExtensions.kt
+- ...
 ```
 
-#Object Expressions and Declarations
+# Object Expressions and Declarations
 
-##Object expressions
+## Object expressions
 
-To avoid too long method, we should externalize object expressions as properties
-:
+To avoid writing methods that are too long, externalize object expressions as
+properties:
```

Comment:
maybe we should specifies something like "unless you can use a direct lambda expression or if the actual object expression is not so long, like ~5 lines max or something"?